src/upstream_patches_ui.sh

```diff
@@ -113,14 +113,19 @@ function dashboard_entry_menu()
 # developer. For this reason, we just need to display the save patches.
 function show_bookmarked_patches()
 {
+  local fallback_message
+
   if [[ -z "${screen_sequence['RETURNING']}" ]]; then
     get_bookmarked_series bookmarked_series
   fi
 
   # Avoiding stale value
   screen_sequence['RETURNING']=''
 
-  list_patches 'Bookmarked patches' bookmarked_series "${screen_sequence['SHOW_SCREEN']}"
+  fallback_message='kw could not find any bookmarked patches.'$'\n'
```

Comment:
Could you add another `\n` to add a blank line between this message and the next one?